src/benchmark.cr

```diff
@@ -1,3 +1,47 @@
+# A Performance Benchmarking Library
+# Overview
+#
+# The Benchmark module provides methods for benchmarking Ruby code, giving
+# detailed reports on the time taken for each task.
+#
+
+# The Benchmark module provides methods to measure and report the time
+# used to execute Ruby code.
+#
+# * Measure the time to construct the string given by the expression
+#   <code>"a"*1_000_000_000</code>:
+#
+#       require 'benchmark'
```

Comment:
Did you try your sample code?  Crystal doesn't have `'...'` string literal.  This code won't be compiled.